docs/hyn/5.2/identification.md

```diff
@@ -38,3 +38,29 @@ public function boot()
 
 > Be wary about switching tenants. I recommend switching only once during code
 execution. Use a background job to run mass changes on tenant databases.
+
+#### Retrieve current tenant
```

Comment:
One last change 😊 

The 4 hashes is too much; it needs to be one 👍

Reply:
Done